kalite/distributed/static/js/distributed/topics/router.js

```diff
@@ -41,7 +41,7 @@ ChannelRouter = Backbone.Router.extend({
         var current_url = Backbone.history.getFragment();
         var fragments = current_url.split("/").slice(0,-1);
         if (fragments.length > 0) {
-            this.navigate(fragments.slice(0,-1).join("/") + "/");
+            this.navigate(fragments.slice(0,-1).join("/") + "/", {trigger: true});
```

Comment:
I remember when we added this, so that your back button (which uses this function as a callback) works as intended... Does this change the behavior of existing uses of `url_back`?